spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/inbound/lifecycle/InboundConnectorManager.java

```diff
@@ -24,14 +24,7 @@
 import io.camunda.connector.runtime.util.inbound.InboundConnectorFactory;
 import io.camunda.connector.runtime.util.inbound.correlation.InboundCorrelationHandler;
 import io.camunda.operate.dto.ProcessDefinition;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.ConcurrentHashMap;
+import java.util.*;
```

Comment:
Hey, sorry for the late review. We [use](https://github.com/camunda/team-connectors/blob/main/development/CODE_STYLE.md) Google Code Style. So, we try to avoid wildcard imports as per [3.3.1](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).